### DISCUSSION TITLE
Seurat Dataset Comparison

### DISCUSSION THREAD

**Discussion by aoifgri** [satijalab/seurat#4071] [Q&A] [answered]
Hello! I am working with a healthy dataset and a disease dataset, where the disease dataset has more than 3 times the amount of cells compared to the healthy control. What would be the best way to approach this comparison, and how could I integrate them to perform differential gene expression analysis?

**Comment by timoast** [comment, marked as answer]
Please see the introductory data integration vignette on the Seurat website: https://satijalab.org/seurat/articles/integration_introduction.html
